src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 1; i < 4; i++) {
+            Lottery lottery = new Lottery();
+            lottery.getRandomBall();
+            System.out.println("Ball " + i + ":" + "\n" + lottery.getRandomBall());
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println("Ball " + (i + 1) + ":" + "\n" + lottery.getRandomBall());
        }
```
Programmers count from 0))
lets create Lottery object outside the for-loop - this way you will create just 1 object of Lottery instead of 3